tests/publiccloud/bsc_1205002.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test to check for bsc#1205002
+#
+# Maintainer: <qa-c@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::ec2;
+use publiccloud::ssh_interactive "select_host_console";
+
+sub run {
+    my ($self, $args) = @_;
+    select_host_console();    # select console on the host, not the PC instance
+
+    my $provider = $self->provider_factory();
```

Comment:
I have a question about this line , but to ask it I need to see how and where you will call this module . 
In short - you can not just call `provider_factory()`  you need also to call `create_instance()`  or maybe you creating instance in another module and getting it here via `$args` variable but than you don't need to call `provider_factory` either

Reply:
yes , now all is fine